src/main/java/com/bookstore/bookstoreapi/data/entity/Rating.java

```diff
@@ -0,0 +1,41 @@
+package com.bookstore.bookstoreapi.data.entity;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Entity
+@Table(name = "Rating")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Rating {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "RatingID")
+    private Long id;
+
+    @Column(name = "Book")
+    private Book book;

```

Comment:
This column actually needs to be a foreign key so you can retrieve data from book entity (the BookID).
The relationship for book should IMO be @ManyToOne, (Don't forget that you need a reverse relationship in the Book entity class that is @OneToMany).